jack-core/src/com/rapleaf/jack/transaction/TransactorImpl.java

```diff
@@ -25,8 +25,15 @@
 
   private final IDbManager<DB> dbManager;
 
-  TransactorImpl(IDbManager<DB> dbManager) {
+  private static final int queryLogSize = 10;
```

Comment:
Change the name of variableto upper camel case. This is the naming convention for static fields.

Also bump it to a larger number, like 30.